### DISCUSSION TITLE
Plot chart with connecting lines?

### DISCUSSION THREAD

**Discussion by halloleo** [saulpw/visidata#1595] [Q&A] [answered]
I'm trying to plot stuff for the first time. 😄

The scatter plots are very helpful, but sometimes a _line_ plot, i.e. a scatter plot with lines connecting the dots, would be great.

In the [API](https://www.visidata.org/docs/graphics/) I saw that something like line graphs exist, but in the [doco](https://www.visidata.org/docs/graph/) I cannot find it.

Do I miss something?

**Comment by saulpw** [comment, marked as answer]
Hi @halloleo, there aren't any line plots available, only the scatter plot that you know already.  For Shapefiles and GeoJSON and TTF files, there are polylines and bezier curves, but they can't be used on graphs currently.

**Reply by halloleo** [reply]
Thx Saul. Just checking whether I missed something.

I was thinking *maybe* to have stab at a plugin which outputs a simple mathplotlib graph in a GUI window...
